modules/optimeraBidAdapter.js

```diff
@@ -0,0 +1,88 @@
+import {registerBidder} from 'src/adapters/bidderFactory';
+const BIDDER_CODE = 'optimera';
+const SCORES_BASE_URL = 'https://s3.amazonaws.com/elasticbeanstalk-us-east-1-397719490216/json/client/';
+
+export const spec = {
+  code: BIDDER_CODE,
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {bidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bidRequest) {
+    if (typeof bidRequest.params.custom != 'undefined' && typeof bidRequest.params.custom.clientID != 'undefined') {
+      return true;
+    } else {
+      return false;
+    }
+  },
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {validBidRequests[]} - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function (validBidRequests) {
+    var optimeraHost = window.location.host;
+    var optimeraPathName = window.location.pathname;
+    var timestamp = Math.round(new Date().getTime() / 1000);
+    var oDv = [];
+    if (typeof validBidRequests[0].params.custom.clientID != 'undefined') {
+      var clientID = validBidRequests[0].params.custom.clientID;
+      oDv.push(clientID);
+      for (var i = 0; i < validBidRequests.length; i++) {
+        oDv.push(validBidRequests[i].adUnitCode);
+      }
+      window.oDv = oDv;
```

Comment:
If your adapter is dependent on this global, then your adapter will not work with concurrent auctions (which is a requirement of 1.0).

Reply:
No problem. This is out and we can do it elsewhere.